.pipelines/templates/.build-and-test-windows-vhd-template.yaml

```diff
@@ -49,6 +49,14 @@ parameters:
   - name: windowsCoreImageUrl
     displayName: Windows core base container image URL Override
     type: string
+  - name: windowsContainerImageJsonUrl
+    displayName: Windows container image JSON URL Override
+    type: string
+    default: https://wcctagentbakerstorage.blob.core.windows.net/simship/2025/07B/payload.json

```

Comment:
Defaul to empty string, "", as this won't be used in the UI which cases the issues